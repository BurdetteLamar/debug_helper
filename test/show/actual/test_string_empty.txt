---
String (name='test_string_empty' size=0 encoding=UTF-8 ascii_only=true bytesize=0):
- ''
