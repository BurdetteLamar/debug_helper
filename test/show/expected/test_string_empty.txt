---
String (message='test_string_empty'):
  to_s: ''
  size: 0
  encoding: !ruby/encoding UTF-8
  ascii_only?: true
  bytesize: 0
